server/pkg/builtin/manifest_ja.yml

```
@@ -1405,4 +1406,21 @@ extensions:
+        title: リンクボタン
+        fields:
+          title:
+            title: タイトル
+          color:
+            title: 色
+          bgColor:
+            title: 背景色
+          url:
+            title: リンクURL
+
+  
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)